lints/lint_ext_ian_no_entries.go

```diff
@@ -30,7 +32,7 @@ func (l *IANNoEntry) CheckApplies(c *x509.Certificate) bool {
 
 func (l *IANNoEntry) Execute(c *x509.Certificate) *LintResult {
 	ian := util.GetExtFromCert(c, util.IssuerAlternateNameOID)
-	if (ian.Value)[1] == 0 {
+	if len(ian.Value) < 2 || bytes.Equal(ian.Value, []byte{0x30, 0x00}) { // if ian is an empty octet string (len == 0 ) or an empty sequence ( 0x30 0x00 ) return error
```

Comment:
I think it might make sense to extract this out into a helper function given that it's used in a few places?

Reply:
Yeah, could make a helper IsEmptySequence()

Reply:
Sounds good, this will be good to merge after that change.